Steve Papacharalambous <stevep@freescale.com> 28 September2006

Patch to fix build errors when building with make 3.81
Error fixed by this patch:
[snip]
make[1]: Entering directory `/home/stevep/work/ltib/rpm/BUILD/e2fsprogs-1.34/po'/bin/sh: -c: line 4: syntax error: unexpected end of file
make[1]: *** [cs.po] Error 2
make[1]: Leaving directory `/home/stevep/work/ltib/rpm/BUILD/e2fsprogs-1.34/po'
make: *** [all-progs-recursive] Error 1
error: Bad exit status from /home/stevep/work/ltib/tmp/rpm-tmp.70816 (%build)
[/snip]

Patch originated by Steve Papacharalambous.

diff --exclude CVS -uNr e2fsprogs-1.34/po/Makefile.in.in e2fsprogs-1.34.modified/po/Makefile.in.in
--- e2fsprogs-1.34/po/Makefile.in.in	Thu Sep 28 16:11:15 2006
+++ e2fsprogs-1.34.modified/po/Makefile.in.in	Thu Sep 28 16:11:01 2006
@@ -108,7 +108,7 @@
 $(POFILES): $(srcdir)/$(DOMAIN).pot
 	@lang=`echo $@ | sed -e 's,.*/,,' -e 's/\.po$$//'`; \
 	test "$(srcdir)" = . && cdcmd="" || cdcmd="cd $(srcdir) && "; \
-	echo "$${cdcmd}$(MSGMERGE_UPDATE) $${lang}.po $(DOMAIN).pot"; \
+	echo "$${cdcmd}$(MSGMERGE_UPDATE) $${lang}.po $(DOMAIN).pot";
 
 
 install: install-exec install-data
